tests/backup/backup_test.go

```diff
@@ -549,6 +549,118 @@ var _ = Describe("{BasicBackupCreation}", func() {
+		appContexts       []*scheduler.Context
+		bkpNamespaces     []string
+		clusterUid        string
+		clusterStatus     api.ClusterInfo_StatusInfo_Status
+		restoreName       string
+		cloudCredName     string
+		cloudCredUID      string
+		backupLocationUID string
+		bkpLocationName   string
+		numDeployments    int
+		providers         []string
+	)
+	backupLocationMap := make(map[string]string)
+	labelSelectors := make(map[string]string)
+	bkpNamespaces = make([]string, 0)
+	selectedBkpNamespaces := make([]string, 0)
+	namespaceMapping := make(map[string]string)
+	JustBeforeEach(func() {
+		numDeployments = 6 // For this test case to have relevance, it is necessary to raise the number of deployments.
+		providers = getProviders()
+
+		StartTorpedoTest("BasicSelectiveRestore", "All namespace backup and restore selective namespaces", nil, 83717)
+		log.InfoD("Deploy applications")
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < numDeployments; i++ {
+			taskName := fmt.Sprintf("%s-%d", taskNamePrefix, i)
+			appContexts = ScheduleApplications(taskName)
+			contexts = append(contexts, appContexts...)
+			for _, ctx := range appContexts {
+				ctx.ReadinessTimeout = appReadinessTimeout
+				namespace := GetAppNamespace(ctx, taskName)
+				bkpNamespaces = append(bkpNamespaces, namespace)
+			}
+		}
+		selectedBkpNamespaces, _ = GetSubsetSlice(bkpNamespaces, len(bkpNamespaces)/2)
+		log.InfoD("Selected application namespaces to restore: %#v", selectedBkpNamespaces)
+	})
+	It("Selective Restore From A Basic Backup", func() {
+
+		Step("Validate applications", func() {
+			ValidateApplications(contexts)
+		})
+		Step("Creating backup location and cloud setting", func() {
+			log.InfoD("Creating bucket, backup location and cloud setting")
+			for _, provider := range providers {
+				cloudCredName = fmt.Sprintf("%s-%s-%v", "cred", provider, time.Now().Unix())
+				bkpLocationName = fmt.Sprintf("%s-%s-bl", provider, getGlobalBucketName(provider))
+				cloudCredUID = uuid.New()
+				backupLocationUID = uuid.New()
+				backupLocationMap[backupLocationUID] = bkpLocationName
+				CreateCloudCredential(provider, cloudCredName, cloudCredUID, orgID)
+				err := CreateBackupLocation(provider, bkpLocationName, backupLocationUID, cloudCredName, cloudCredUID, getGlobalBucketName(provider), orgID, "")
+				dash.VerifyFatal(err, nil, "Creating backup location")
+			}
+		})
+		Step("Register cluster for backup", func() {
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			err = CreateSourceAndDestClusters(orgID, "", "", ctx)
+			dash.VerifyFatal(err, nil, "Creating source and destination cluster")
+			clusterStatus, clusterUid = Inst().Backup.RegisterBackupCluster(orgID, SourceClusterName, "")
+			dash.VerifyFatal(clusterStatus, api.ClusterInfo_StatusInfo_Online, fmt.Sprintf("Verifying backup cluster with uid: %s", clusterUid))
+		})
+		Step("Taking backup of applications", func() {
```

Comment:
Change the step to `Taking backup of multiple namespaces`

Reply:
Changed the step and the following log description to "Taking backup of multiple namespaces".